chrome/content/quicktext.js

```diff
@@ -1,7 +1,8 @@
-var gQuicktext = Components.classes["@hesslow.se/quicktext/main;1"].getService(Components.interfaces.wzIQuicktext);
-var gQuicktextVar = Components.classes["@hesslow.se/quicktext/variables;1"].createInstance(Components.interfaces.wzIQuicktextVar);
+var { gQuicktext } = Components.utils.import("chrome://quicktext/content/components/wzQuicktext.js", null);
+Components.utils.import("chrome://quicktext/content/components/wzQuicktextVar.js");
+var gQuicktextVar = new wzQuicktextVar();
```

Comment:
To prepare for the future, best to use the `var { wzQuicktextVar } = Cu...` syntax everywhere. This will make it easier to transition to ES6 imports.

Reply:
Done.